packages/styles/scss/components/_callout.scss

```diff
@@ -16,42 +17,45 @@
     max-height: 300px;
     overflow: hidden;
 
-    #{$self}--toggle--icon {
+    #{$callout}--toggle__icon {
```

Comment:
This doesn't follow the standard. Should be something like `#{$callout}__toggle-icon`